src/site/paragraphs/linkTeaser.drupal.liquid

```diff
@@ -33,16 +33,20 @@
   data-links-list-header="{{ link.title | escape }}"
   data-links-list-section-header="{{ section_header | escape }}"
   {% if parentFieldName === 'field_spokes' %}class="hub-page-link-list__item"{% endif %}>
-    <a href="{{link.url.path}}"
-      {% if linkTeaser.options["target"] %}target="{{ linkTeaser.options["target"] }}"{% endif %}
-      >
-        {% if link.title != empty %}
-            <{{ header }} class="{{ headerClass }}">{{ link.title }}</{{ header }}>
-            {% if parentFieldName === "field_spokes" %}
-                <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
-            {% endif %}
-        {% endif %}
-    </a>
+    {% if link.title != empty %}
+      <{{ header }} class="{{ headerClass }}">
+        <a href="{{link.url.path}}" class="vads-u-text-decoration--underline"
+          {% if linkTeaser.options["target"] %}
+            target="{{ linkTeaser.options["target"] }}"
```

Comment:
If this `target` has the potential to be `_blank` we need to announce the to a screen reader and also add `rel="noopener noreferrer"` some more info on that: https://timwright.org/blog/2017/01/21/opening-links/

Reply:
I wish ESLint rules applied to `.liquid` files. Specifically, the `jsx-no-target-blank` rule: 
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

Reply:
It would be if we used gatsby!

Reply:
Thanks for the link to your blog, learned something new! Updated 💯